MODULE.bazel

```diff
@@ -14,6 +14,7 @@ bazel_dep(name = "bazel_skylib", version = "1.6.1")
 # Required by @remote_java_tools, which is loaded via module extension.
 bazel_dep(name = "rules_proto", version = "4.0.0")
 bazel_dep(name = "rules_license", version = "0.0.3")
+bazel_dep(name = "abseil-cpp", version = "20240116.2", repo_name = "com_google_absl")
```

Comment:
@fmeum Why was this new dep required in this change?

Reply:
This is a dep of oneversion when built from source.

Reply:
I see, thanks.